api/v1beta1/common.go

```diff
@@ -63,6 +63,6 @@ func validateIBMPowerVSResourceReference(res IBMPowerVSResourceReference, resTyp
 
 func defaultIBMVPCMachineSpec(spec *IBMVPCMachineSpec) {
 	if spec.Profile == "" {
-		spec.Profile = "bx2-4x16"
+		spec.Profile = "bx2-2x8"
```

Comment:
Though there are few more profiles lower than this but in the balanced one this is the bare minimal one per [info](https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui), will revisit this later if any more concerns..

/unhold